services/subscriber/service_test.go

```diff
@@ -185,7 +185,6 @@ func TestService_ModeALL(t *testing.T) {
 }
 
 func TestService_ModeANY(t *testing.T) {
-	t.Skip("TODO: flaky test.")
```

Comment:
Is this implication here that you think moving to this new infrastructure will fix this flaky test?